src/main/java/org/apache/commons/crypto/jna/OpenSslJnaCryptoRandom.java

```diff
@@ -103,10 +103,9 @@ public void nextBytes(byte[] bytes) {
             //to support multithreading https://wiki.openssl.org/index.php/Manual:Threads(3) needs to be done
             
             if(rdrandEnabled && OpenSslNativeJna.RAND_get_rand_method().equals(OpenSslNativeJna.RAND_SSLeay())) {
-                close();
```

Comment:
Why do you need to remove this?

Reply:
I don't remember, but I added it back and tests pass fine.